GooglePay/src/main/java/com/braintreepayments/api/GooglePayClient.java

```diff
@@ -97,25 +97,24 @@ public void isReadyToPay(@NonNull final FragmentActivity activity,
         try {
             Class.forName(PaymentsClient.class.getName());
         } catch (ClassNotFoundException | NoClassDefFoundError e) {
-            callback.onResult(false, null);
+            callback.onGooglePayReadinessResult(new GooglePayReadinessResult.NotReadyToPay(null));
```

Comment:
```suggestion
            callback.onGooglePayReadinessResult(new GooglePayReadinessResult.NotReadyToPay(e));
```
We could forward the cause here, or wrap it in a `BraintreeExcpetion`.

Reply:
Good catch